src/pl/senla/hotel/controller/ConsoleCollection.java

```diff
@@ -3,92 +3,210 @@
 import pl.senla.hotel.entity.*;
 
 import java.time.LocalDate;
+import java.time.LocalDateTime;
 import java.util.ArrayList;
 import java.util.List;
 
 import static pl.senla.hotel.constant.ConsoleConstant.*;
+import static pl.senla.hotel.constant.HotelConstant.*;
 
 public class ConsoleCollection implements Console{
 
-    //Было так. Сделать после всех проверок внедрение зависимостей через конструктор
-    private final Controller<Room> roomController = new RoomController();
-    private final Controller<RoomReservation> roomReservationController = new RoomReservationController();
-    private final Controller<Client> clientController = new ClientController();
-    private final Controller<Order> orderController = new OrderController();
+    private final ControllerRoom roomController = new ControllerRoomCollection();
+    private final ControllerRoomReservation roomReservationController = new ControllerRoomReservationCollection();
+    private final ControllerGuest guestController = new ControllerGuestCollection();
+    private final ControllerOrder orderController = new ControllerOrderCollection();
 
     @Override
     public void start(){
 
-        System.out.println("\n----- Room -----");
-
-        Room room0 = new Room(0,11,100,1, RoomLevel.STANDART.getLevel(), RoomStatus.FREE.getStatus());
-        Room room1 = new Room(1,12,120,2,RoomLevel.ECONOM.getLevel(), RoomStatus.FREE.getStatus());
-        Room room2 = new Room(2,13,130,2,RoomLevel.LUX.getLevel(), RoomStatus.FREE.getStatus());
+        System.out.println("\n----- Room and FreeRoom -----");
+        System.out.println("==============================");
+        System.out.println("CREATE and READ ROOMS");
```

Comment:
Let's reduce amount of basic entities (guest, room) to 4-5, it will also reduce dependent entities. 
It's to much right now, the are similar but create a wall of code and logs

Reply:
Or if you want have all current data, try to optimize code (streams, lambdas, switches) to make it less duplicated

Reply:
I used only these basic Entities:
1. Room
2. Guest
3. RoomReservation (it is connected with Room and Guest)
4. FreeRoom (it's the opposite of a RoomReservation and will be used for checking if the creating of new RoomReservation is available). My opinion: It's impossible to delete this entity.
5. HotelService (it's parent/abstract class for all types of Hotel Services and is used for creating of List of all Guest's Services in Order).
6. Order (it serves to connection of all different types of Orders which Guest reserved).
------------------------
7 and 8. Restaurant and GuideTour - these HotelServices were created as examples of other HotelServices but I don't use them.

_**"Or if you want have all current data, try to optimize code (streams, lambdas, switches) to make it less duplicated."**_
ok. I use them...

Should I use Builder for creating new instances?

Reply:
No no, entities are ok, I meant amount of created entities, you have more then 10 room and more then 10 guests. Let's reduce it to 4-5. It's to much to read it in code and logs

Reply:
You can use builders if you understand them. Right now it's not necessary, but totally acceptable